src/stateToHTML.js

```diff
@@ -284,16 +290,26 @@ class MarkupGenerator {
     }
   }
 
-  openWrapperTag(wrapperTag: string, customRendererOptions) {
+  openWrapperTag(wrapperTag: string, customRendererOptions, blockData) {
```

Comment:
A third argument, especially when the second one is optional is problematic. Can you change the function to accept two arguments, when the second one is an object with `customRendererOptions` `blockData` optional properties?